File: 	1049.js
var input = require('fs').readFileSync('/dev/stdin', 'utf8');
var lines = input.split('\n');

let X = lines.shift();
let Y = lines.shift();
let Z = lines.shift();

if (X == 'vertebrado' & Y == 'ave' & Z =='carnivoro') {
console.log('aguia');
}
if (X == 'vertebrado' & Y =='ave' & Z == 'onivoro') {
 console.log('pomba');
}
if (X == 'vertebrado' & Y =='mamifero' & Z == 'onivoro') {
  console.log('homem');
}
if (X =='vertebrado' & Y =='mamifero' & Z =='herbivoro') {
console.log('vaca');
}
if (X =='invertebrado' & Y =='inseto' & Z == 'hematofago') {
    console.log('pulga');
}
if (X =='invertebrado' & Y =='inseto' & Z == 'herbivoro') {
 console.log('lagarta');
}
if (X =='invertebrado'  & Y == 'anelideo'  & Z == 'hematofago'){
console.log('sanguessuga');
}
if (X =='invertebrado' & Y == 'anelideo' & Z =='onivoro') {
  console.log('minhoca');
}

File: 	1049.java
import java.util.Scanner;

public class Main {

	public static void main(String[] args) {
		Scanner sc = new Scanner(System.in);

		String X = sc.nextLine();
		String Y = sc.nextLine();
		String Z = sc.nextLine();
		
		if (X.equals("vertebrado") & Y.equals( "ave") & Z.equals( "carnivoro")) {
			System.out.println("aguia");
		}
		if (X.equals( "vertebrado") & Y.equals("ave") & Z.equals( "onivoro")) {
			System.out.println("pomba");
		}
		if (X.equals( "vertebrado") & Y.equals ("mamifero") & Z.equals( "onivoro")) {
			System.out.println("homem");
		}
		if (X.equals( "vertebrado") & Y.equals( "mamifero") & Z.equals( "herbivoro")) {
			System.out.println("vaca");
		}
		if (X.equals( "invertebrado") & Y.equals("inseto") & Z.equals( "hematofago")) {
			System.out.println("pulga");
		}
		if (X.equals( "invertebrado") & Y.equals("inseto") & Z.equals( "herbivoro")) {
			System.out.println("lagarta");
		}
		if (X.equals("invertebrado" ) & Y.equals( "anelideo" ) & Z.equals( "hematofago") ){
			System.out.println("sanguessuga");
		}
		if (X.equals("invertebrado") & Y.equals( "anelideo") & Z.equals("onivoro")) {
			System.out.println("minhoca");
		}
	}

}
